### PR TITLE
Added Tailwindcss Docs Plugin

### PR DESCRIPTION
**Pull request by areeburrub**

### CODE REVIEW

File: plugins.json [hunk 1]
```diff
@@ -13,6 +13,18 @@
...
+        "ID": "5c221cee-07a5-4e4c-a669-a0e0f7486c65",  
+        "Name": "Tailwindcss",
+        "Description": "Search tailwindcss Docs",
+        "Author": "Areeb ur Rub",
+        "Version": "1.0.0",
+        "Language": "executable",
+        "Website": "https://github.com/areeburrub/tailwindcss-flow-launcher-plugin",
+        "UrlDownload": "https://github.com/areeburrub/tailwindcss-flow-launcher-plugin/releases/download/v1.0.0/tailwindcss-plugin-flow-launcher.zip",
+        "UrlSourceCode": "https://github.com/areeburrub/tailwindcss-flow-launcher-plugin/tree/main",
+        "IcoPath": "https://raw.githubusercontent.com/areeburrub/tailwindcss-flow-launcher-plugin/main/app.png",
```
Comment (jjw24) [inline plugins.json:26]:
```suggestion
        "IcoPath": "https://raw.githubusercontent.com/areeburrub/tailwindcss-flow-launcher-plugin/main/app.png"
```

File: plugins.json [hunk 2]
```diff
@@ -13,6 +13,18 @@
...
+        "ID": "5c221cee-07a5-4e4c-a669-a0e0f7486c65",  
+        "Name": "Tailwindcss",
+        "Description": "Search tailwindcss Docs",
+        "Author": "Areeb ur Rub",
+        "Version": "1.0.0",
+        "Language": "executable",
+        "Website": "https://github.com/areeburrub/tailwindcss-flow-launcher-plugin",
+        "UrlDownload": "https://github.com/areeburrub/tailwindcss-flow-launcher-plugin/releases/download/v1.0.0/tailwindcss-plugin-flow-launcher.zip",
+        "UrlSourceCode": "https://github.com/areeburrub/tailwindcss-flow-launcher-plugin/tree/main",
+        "IcoPath": "https://raw.githubusercontent.com/areeburrub/tailwindcss-flow-launcher-plugin/main/app.png"
```
Comment (jjw24) [inline plugins.json:26]:
```suggestion
        "IcoPath": "https://cdn.jsdelivr.net/gh/areeburrub/tailwindcss-flow-launcher-plugin/app.png"
```

File: plugins.json [hunk 3]
```diff
@@ -13,6 +13,18 @@
         "LatestReleaseDate": "2022-10-04T21:31:59Z",
         "DateAdded": "2022-10-08T16:26:57Z"
     },
+    {
+        "ID": "5c221cee-07a5-4e4c-a669-a0e0f7486c65",  
+        "Name": "Tailwindcss",
+        "Description": "Search tailwindcss Docs",
+        "Author": "Areeb ur Rub",
+        "Version": "1.0.0",
+        "Language": "executable",
```
Comment (jjw24) [inline plugins.json:22]:
```suggestion
        "Language": "typescript",
```

**Comment by jjw24** [comment]
Hi @areeburrub, I updated the plugin to use Flow's Node.js setup https://github.com/areeburrub/tailwindcss-flow-launcher-plugin/pull/1, please review and merge. Once done I will merge this PR in.

**Comment by areeburrub** [comment]
Okay I have merged your changes

**Review by jjw24 — APPROVED** [review]

**Pull request merged**